### ISSUE TITLE
Error: /dir/ is not mounted or bad option.

### ISSUE THREAD

**Issue by gergesh** [dnschneid/crouton#2776]
```
$ sudo edit-chroot -all
No chroots found in /mnt/stateful_partition/crouton/chroots
$ sudo edit-chroot -all -c /media/removable/SD\ Card/chroots
name: kali-rolling
encrypted: no
Entering /media/removable/SD Card/chroots/kali-rolling...
mount: /run/crouton/media/removable/SD Card/chroots/kali-rolling/sys/fs/selinux not mounted or bad option

       In some cases useful info is found in syslog - try
       dmesg | tail or so.
```

#### Please describe your issue:
Can't enter the chroot.
dmesg | tail output:
```

[13477.374358] wlan0: associate with MAC Address (try 1/3)
[13477.375272] wlan0: RX AssocResp from MAC Address (capab=0x431 status=0 aid=1)
[13477.376917] wlan0: associated
[13477.376955] IPv6: ADDRCONF(NETDEV_CHANGE): wlan0: link becomes ready
[13479.581646] perf samples too long (10015 > 10000), lowering kernel.perf_event_max_sample_rate to 13000
[14188.428915] EXT4-fs (sdb): re-mounted. Opts: data=ordered
[14415.477498] tpm_tis tpm_tis: command 0x65 (size 20) returned code 0x0
[14415.507461] tpm_tis tpm_tis: command 0x65 (size 22) returned code 0x0
[14415.537750] tpm_tis tpm_tis: command 0x65 (size 22) returned code 0x0
[14426.582408] EXT4-fs (sdb): re-mounted. Opts: data=ordered
```
System info:
Toshiba Chromebook 2 2015 3215U
Chrome Version 55.0.2858.0 dev (64-bit)
crouton version 1-20160902144033~master:9f8e9a22
When entering /SD Card/chroots/kali-rolling/sys/fs/selinux through the file manager, it's empty.

**Comment by gergesh** [comment]
Hmm, not sure if that's what fixed it but putting it out there just in case. I've given up on the external chroot today and installed kali-rolling to the internal storage. Suddenly, I can chroot with no problems at all (to both of them).

**Comment by DennisLfromGA** [comment]
I've had luck with first installing and setting up my chroots on internal storage and then backing up and restoring them to an external drive. Once it's all working on the external drive, you can remove the chroot from your Chromebook. It's always good to have a backup too.

**Comment by drinkcat** [comment]
Probably fixed by 9f8e9a2253f12eef4040c49dce23c352bad9e2ac, please reopen if this is still an issue.
